client/app/instructor/events/events.html

```diff
@@ -19,7 +19,14 @@ <h1 class="page-header">Events</h1>
                 </div>
               </div>
               <div>
-                {{event.info.description}}
+                {{event.description}}
+              </div>
+              <div class="event-section">
+                <ul>
+                  <li ng-repeat="sectionEvent in event.sections" ng-click='goToEvent(sectionEvent)'>
+                    <b>Course:</b> {{sectionEvent.course}}, <b>{{(sectionEvent.sectionNumbers.length===1)?"Section: ":"Sections: "}}</b> <span ng-repeat="num in sectionEvent.sectionNumbers"> {{num}}<span ng-hide="$last">, </span></span>
```

Comment:
It looks to me like the essential info is all there, and it's not annoying to use (especially once we have an events page so that clicking on the event goes somewhere).

Formatting changes should be done, like #161 and #219, but can be put off for now.